---
title: Citing Matchmaker Exchange
---
{% include header.html %}
<div class='container-fluid  bg-grey'>
<div class='container'>
  <div class='row-fluid clearfix padding-bottom-lg padding-top-sm'>
    <div class='col-sm-12 col-xs-12'>
      <a id="top" name="top"></a>
      <h1 class='text-white'>Citing Matchmaker Exchange</h1>
    </div>
  </div>
</div>
</div>
<div class='container'>
  <div class='row-fluid clearfix padding-bottom-lg padding-top-lg'>
    <div class='col-sm-12 col-xs-12'>
      <p>Please acknowledge the Matchmaker Exchange, and the specific MME node that supported any discoveries, in publications. The table below provides the citations to use for Matchmaker Exchange and the individual Matchmaker Exchange nodes.</p>


      <h3 class="padding-top-lg">Citing the Matchmaker Exchange</h3>

      <blockquote class="text-sm">
      Philippakis AA, Azzariti DR, Beltran S, Brookes AJ, Brownstein CA, Brudno M, Brunner HG, Buske OJ, Carey K, Doll C, Dumitriu S, Dyke SOM, den Dunnen JT, Firth HV, Gibbs RA, Girdea M, Gonzalez M, Haendel MA, Hamosh A, Holm IA, Huang L, Hurles ME, Hutton B, Krier JB, Misyura A, Mungall CJ, Paschall J, Paten B, Robinson PN, Schiettecatte F, Sobreira NL, Swaminathan GJ, Taschner PE, Terry SF, Washington NL, Züchner S, Boycott KM, Rehm HL. 2015. <a href="http://onlinelibrary.wiley.com/doi/10.1002/humu.22858/full">The Matchmaker Exchange: A Platform for Rare Disease Gene Discovery</a>. <em>Human Mutation</em>, <strong>36</strong>: 915–921. doi:<a href="http://dx.doi.org/10.1002/humu.22858/">10.1002/humu.22858</a>

      [<a href="assets/files/philippakis2015matchmaker.bib">BibTex</a>]
      </blockquote>

      <h3 class="padding-top-lg">Please cite any MME node that contributed to your match</h3>

<table class="table table-striped table-bordered table-responsive">
  <thead>
    <tr>
      <td class="info text-sm text-tight">Name</td>
      <td class="info text-sm text-tight">Primary Reference</td>
      <td class="info text-sm text-tight">Primary URL</td>
      <td class="info text-sm text-tight">Notes</td>
    </tr>
  </thead>
  <tr>
    <td>DECIPHER</td>
    <td>DECIPHER: Database of Chromosomal Imbalance and Phenotype in Humans using Ensembl Resources. Firth, H.V. et al (2009). Am.J.Hum.Genet 84, 524-533 (DOI: dx.doi.org/10/1016/j.ajhg.2009.03.010)</td>
    <td><a href="https://decipher.sanger.ac.uk/">https://decipher.sanger.ac.uk/</a></td>
    <td>Acknowledgement for publication: “This study makes use of data generated by the DECIPHER community. A full list of centres who contributed to the generation of the data is available from http://decipher.sanger.ac.uk and via email from decipher@sanger.ac.uk. Funding for the project was provided by Wellcome.” https://decipher.sanger.ac.uk/about/citing</td>
  </tr>
  <tr>
    <td>GeneMatcher</td>
    <td><ol><li>Sobreira, N., Schiettecatte, F., Valle, D. and Hamosh, A. (2015), GeneMatcher: A Matching Tool for Connecting Investigators with an Interest in the Same Gene. Human Mutation, 36: 928–930.</li><li>Sobreira N, Schiettecatte F, Boehm C, Valle D, Hamosh A. New tools for Mendelian disease gene identification: PhenoDB variant analysis module; and GeneMatcher, a web-based tool for linking investigators with an interest in the same gene. Hum Mutat. 2015 Apr;36(4):425-31. doi: 10.1002/humu.22769. PubMed: 25684268.</li></ol></td>
    <td><a href="https://www.genematcher.org/">https://www.genematcher.org/</a></td>
    <td>&nbsp;</td>
  </tr>
  <tr>
    <td>IRUD</td>
    <td>Adachi T, Kawamura K, Furusawa Y, Nishizaki Y, Imanishi N, Umehara S, Izumi K, Suematsu M. Japan's initiative on rare and undiagnosed diseases (IRUD): towards an end to the diagnostic odyssey. Eur J Hum Genet. 2017 Sep;25(9):1025-1028. doi: 10.1038/ejhg.2017.106. PMID: 28794428; PMCID: PMC5558173.</td>
    <td>&nbsp;</td>
    <td>Acknowledgement for publication: This study makes use of data shared through Initiative on Rare and Undiagnosed Diseases (IRUD). This study makes use of data shared through the IRUD Exchange repository. Funding for IRUD was provided by Japan Agency for Medical Research and Development (AMED)</td>
  </tr>
  <tr>
    <td>ModelMatcher</td>
    <td>Harnish JM, Li L, Rogic S, Poirier-Morency G, Kim SY; Undiagnosed Diseases Network, Boycott KM, Wangler MF, Bellen HJ, Hieter P, Pavlidis P, Liu Z, Yamamoto S. (2022) ModelMatcher: A scientist-centric online platform to facilitate collaborations between stakeholders of rare and undiagnosed disease research. Hum Mutat. 2022 Feb 27. doi: 10.1002/humu.24364. Online ahead of print. PMID: 35224820</td>
    <td>https://www.modelmatcher.net/</td>
    <td>Acknowledgement for publication: The ModelMatcher project is supported by the Jan and Dan Duncan Neurological Institute at Texas Children's Hospital and by a grant from the National Institutes of Health (NIH, U54NS093793).</td>
  </tr>

  <tr>
    <td>Monarch Initiative</td>
    <td>Mungall CJ, McMurry JA, Kohler S, Balhoff JP, Borromeo C, Brush M, Carbon S, Conlin T, Dunn N, Engelstad M, Foster E, Gourdine JP, Jacobsen JOB, Keith D, Laraway B, Lewis SE, Nguyen Xuan J, Shefchek K, Vasilevsky N, Yuan Z, Washington N, Hochheiser H, Groza T, Smedley D, Robinson PN, and Haendel MA. (2016) The Monarch Initiative: an integrative data and analytic platform connecting phenotypes to genotypes across species. Nucleic Acids Research, 2016 1
doi: 10.1093/nar/gkw1128</td>
    <td><a href="https://monarchinitiative.org/">https://monarchinitiative.org/</a></td>
    <td>Acknowledgement for publication: Monarch makes use of integrated data from many clinical and basic resources via funding from the USA National Institutes for Health Office of the Director (#5R24OD011883).</td>
  </tr>
  <tr>
    <td>MyGene2</td>
    <td>MyGene2, NHGRI/NHLBI University of Washington-Center for Mendelian Genomics (UW-CMG), Seattle, WA (URL: http://www.mygene2.org [date (month, yr) accessed].</td>
    <td><a href="https://www.mygene2.org/">https://www.mygene2.org/</a></td>
    <td>Acknowledgement for publication: The authors would like to thank the contributors to MyGene2 and the University of Washington Center for Mendelian Genomics for use of data.</td>
  </tr>
   <tr>
    <td>PatientMatcher</td>
    <td></td>
    <td><a href="https://www.scilifelab.se/facilities/clinical-genomics-stockholm/">https://www.scilifelab.se/facilities/clinical-genomics-stockholm/</a></td>
    <td>Acknowledgement for publication: The authors would like to thank the contributors to PatientMatcher at The Karolinska Institute and University Hospital, Stockholm, Sweden and Clinical Genomics, Science For Life Laboratory, Stockholm Sweden. Funding for PatientMatcher was provided by BigMed (Norwegian Research Council).</td>
  </tr>
  <tr>
    <td>PhenomeCentral</td>
    <td>Buske OJ*, Girdea M*, Dumitriu S, Gallinger B, Hartley T, Trang H, Misyura A, Friedman T, Beaulieu C, Bone WP, Links AE, Washington NL, Haendel MA, Robinson PN, Boerkoel CF, Adams D, Gahl WA, Boycott KM, Brudno M. 2015. PhenomeCentral: A Portal for Phenotypic and Genotypic Matchmaking of Patients with Rare Genetic Diseases. Human Mutation, 36: 931–940. doi:10.1002/humu.22851 </td>
    <td><a href="https://www.phenomecentral.org/">https://www.phenomecentral.org/</a></td>
    <td>Acknowledgement for publication: This study makes use of data shared through the PhenomeCentral repository. Funding for PhenomeCentral was provided by Genome Canada and Canadian Institute of Health Research (CIHR).</td>
  </tr>
    <tr>
    <td>RD-Connect GPAP</td>
    <td><ol><li>Lochmüller, H., Badowska, D.M., Thompson, R. et al. RD-Connect, NeurOmics and EURenOmics: collaborative European initiative for rare diseases. Eur J Hum Genet 26, 778–785 (2018) doi:10.1038/s41431-018-0115-5</li><li>Thompson, R., Johnston, L., Taruscio, D. et al. RD-Connect: An Integrated Platform Connecting Databases, Registries, Biobanks and Clinical Bioinformatics for Rare Disease Research. J GEN INTERN MED 29, 780–787 (2014) doi:10.1007/s11606-014-2908-8</li></ol></td>
    <td><a href="https://platform.rd-connect.eu/">https://platform.rd-connect.eu/</a></td>
    <td>Acknowledgement for publication: Data was analyzed using the RD-Connect Genome-Phenome Analysis Platform, which received funding from EU projects RD-Connect, Solve-RD and EJP-RD (grant numbers FP7 305444, H2020 779257, H2020 825575), Instituto de Salud Carlos III (grant numbers PT13/0001/0044, PT17/0009/0019; Instituto Nacional de Bioinformática, INB) and ELIXIR Implementation Studies.</td>
  </tr>
    <tr>
    <td><i>seqr</i></td>
    <td>Lynn S. Pais, Hana Snow,  Ben Weisburd, Shifa Zhang, Samantha Baxter, Stephanie DiTroia, Emily O’Heir, Eleina England, Katherine Chao, Gabrielle Lemire, Ikeoluwa Osei-Owusu, Grace E. VanNoy, Michael Wilson, Kevin Nguyen, Harindra Arachchi, William Phu, Matthew Solomonson, Stacy Mano, Melanie O’Leary, Alysia Lovgren, Lawrence Babb, Christina Austin-Tse, Heidi L. Rehm, Daniel G. MacArthur,  Anne O’Donnell-Luria seqr : a web-based analysis and collaboration tool for rare disease genomics. doi: https://doi.org/10.1101/2021.10.27.21265326/td>
    <td><a href="https://seqr.broadinstitute.org"/td>
    <td>If Broad Institute contributes to the results being published, the authors must acknowledge Broad Institute using the following wording: "This study makes use of data shared through the seqr platform with funding provided by National Institutes of Health grants R01HG009141 and UM1HG008900."	</td>
  </tr>
</table>


      <h3 class="padding-top-lg">Please cite any Resources (ontologies, algorithms, tools) that supported your discovery</h3>

<table class="table table-striped table-bordered table-responsive">
  <thead>
    <tr>
      <td class="info text-sm text-tight">Name</td>
      <td class="info text-sm text-tight">Primary Reference</td>
      <td class="info text-sm text-tight">Primary URL</td>
      <td class="info text-sm text-tight">Notes</td>
    </tr>
  </thead>
  <tr>
    <td>Human Phenotype Ontology</td>
    <td>Köhler S et al. 2017. The Human Phenotype Ontology in 2017. Nucleic Acids Database Issue, in press</td>
    <td><a href="http://human-phenotype-ontology.org/">http://human-phenotype-ontology.org/</a></td>
    <td>Acknowledgement for publication: The authors would like to thank the very many contributors to the Human Phenotype Ontology, a community resource for human phenotypic knowledge.</td>
  </tr>
  <tr>
    <td>Exomiser</td>
    <td>Smedley D, Jacobsen JO, Jäger M, Köhler S, Holtgrewe M, Schubach M, Siragusa
E, Zemojtel T, Buske OJ, Washington NL, Bone WP, Haendel MA, Robinson PN.
Next-generation diagnostics and disease-gene discovery with the Exomiser. Nat
Protoc. 2015 Dec;10(12):2004-15</td>
    <td>&nbsp;</td>
    <td>&nbsp;</td>
  </tr>
</table>


    </div>
  </div>
</div>
<!--/content-->
{% include footer.html %}
